tests/common.go

```diff
@@ -356,6 +348,66 @@ func ValidateAndDestroy(contexts []*scheduler.Context, opts map[string]bool) {
 	})
 }
 
+// DescribeNamespace takes in the scheduler contexts and describes each object within the test context.
+func DescribeNamespace(contexts []*scheduler.Context) {
+	context(fmt.Sprintf("generating namespace info..."), func() {
+		Step(fmt.Sprintf("Describe Namespace objects for test %s \n", ginkgo.CurrentGinkgoTestDescription().TestText), func() {
+			for _, ctx := range contexts {
+				logrus.Info(Inst().S.Describe(ctx))
+			}
+		})
+	})
+}
+
+// CollectSupport creates a support bundle
+func CollectSupport() {
+	context(fmt.Sprintf("generating support bundle..."), func() {
+		Step(fmt.Sprintf("save all useful logs on each node"), func() {
+			nodes := node.GetWorkerNodes()
+			expect(nodes).NotTo(beEmpty())
+
+			for _, n := range nodes {
+				if !n.IsStorageDriverInstalled {
+					continue
+				}
+
+				logrus.Infof("colleting diags from %s", n.Name)
+				Inst().V.CollectDiags(n)
+
+				journalCmd := fmt.Sprintf("journalctl -l > /var/cores/all_journal_%v.log", time.Now().Format(time.RFC3339))
+				logrus.Infof("saving journal output on %s", n.Name)
+				runCmd(journalCmd, n)
+
+				logrus.Infof("saving portworx journal output on %s", n.Name)
+				runCmd("journalctl -lu portworx* > /var/cores/portworx.log", n)
+
+				logrus.Infof("saving kubelet journal output on %s", n.Name)
+				runCmd("journalctl -lu kubelet* > /var/cores/kubelet.log", n)
+
+				logrus.Infof("saving dmesg output on %s", n.Name)
+				runCmd("dmesg -T > /var/cores/dmesg.log", n)
+
+				logrus.Infof("saving disk list on %s", n.Name)
+				runCmd("lsblk > /var/cores/lsblk.log", n)
+
+				logrus.Infof("saving mount list on %s", n.Name)
+				runCmd("cat /proc/mounts > /var/cores/mounts.log", n)
```

Comment:
since you are using /var/cores folder name in many places I would create a variable for that